docs/user-guide/generate-certificates.md

```diff
@@ -14,32 +41,42 @@ zowe:
   setup:
     certificate:
       type: PKCS12
-      dname:
-        caCommonName: 
-        commonName:
-        orgUnit:
-        org:
-        locality:
-        state:
-        country:
+        directory: /var/zowe/keystore
```

Comment:
The `directory` through `caPassword` should be nested under `pkcs12` ... see https://docs.zowe.org/stable/appendix/zwe_server_command_reference/zwe/init/zwe-init-certificate